config/locales/administrate.fr.yml

```diff
@@ -6,6 +6,8 @@ fr:
       destroy: Supprimer
       edit: Modifier
       show: Détails
+      new: Nouveau
+      back: Arrière
```

Comment:
This doesn't make much sense in the context of a flow. I would go for `Précédent` if the goal is `Previous`, or `Retour` if in a modal.